### PR TITLE
Cheatsheet

### PR DESCRIPTION
**Pull request by petebacondarwin**
Before this PR lands we need to add `@cheatsheetSection` and `@cheatsheetItem` tags to the docs in the Angular 2 project.

### CODE REVIEW

**Comment by petebacondarwin** [comment]
OK, so I have refactored this. Now the cheatsheet sections are in their own files under the modules/angular2/docs/cheatsheet folder. This seems about the right place to keep them - Not too far from or too close to the source - and fairly clean layout. See the updated PR at https://github.com/angular/angular/pull/5125 for an example of how this would look.

File: public/resources/js/cheatsheet.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+{
+  "syntax": "         <p *my-unless=\\\"myExpression\\\">...</p> ",
+  "description": "\n         The * symbol means that the current element will be turned into an embedded template. Equivalent to:\\n<template [my-unless]=\\\"myExpression\\\"><p>...</p></template> ",
+  "bold": "\n         *my-unless"
+},
+{
+  "syntax": "         <p>Card No.: {{cardNumber ",
+  "description": " myCreditCardNumberFormatter}}</p> ",
+  "bold": "\n         Transforms the current value of expression cardNumber via pipe called creditCardNumberFormatter. "
+},
```
Comment (todoubaba) [inline public/resources/js/cheatsheet.json]: Misline?

Comment (petebacondarwin) [inline public/resources/js/cheatsheet.json]: Ignore this file. It needs to be regenerated

File: tools/api-builder/angular.io-package/processors/addJadeDataDocsProcessor.spec.js [hunk 2]
```diff
@@ -24,28 +25,30 @@ describe('addJadeDataDocsProcessor', function() {
...
-          { name: 'Beta', docType: 'class'},
-          { name: 'Alpha', docType: 'class'},
-          { name: 'Gamma', docType: 'class'},
-          { name: 'Nu', docType: 'class'},
-          { name: 'Mu', docType: 'class'}
+          beta = { name: 'Beta', docType: 'class'},
+          alpha = { name: 'Alpha', docType: 'class'},
+          gamma = { name: 'Gamma', docType: 'class'},
+          nu = { name: 'Nu', docType: 'class'},
+          mu = { name: 'Mu', docType: 'class'}
```
Comment (todoubaba) [inline tools/api-builder/angular.io-package/processors/addJadeDataDocsProcessor.spec.js]: beta: {...} ?

**Pull request merged**